config.go

```diff
@@ -3,31 +3,38 @@ package fangs
 import (
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/anchore/go-logger"
 	"github.com/anchore/go-logger/adapter/discard"
 )
 
 type Config struct {
-	Logger  logger.Logger `yaml:"-" json:"-" mapstructure:"-"`
-	AppName string        `yaml:"-" json:"-" mapstructure:"-"`
-	TagName string        `yaml:"-" json:"-" mapstructure:"-"`
-	File    string        `yaml:"-" json:"-" mapstructure:"-"`
-	Finders []Finder      `yaml:"-" json:"-" mapstructure:"-"`
+	Logger                 logger.Logger `yaml:"-" json:"-" mapstructure:"-"`
+	AppName                string        `yaml:"-" json:"-" mapstructure:"-"`
+	TagName                string        `yaml:"-" json:"-" mapstructure:"-"`
+	ConfigureMultipleFiles bool          `yaml:"-" json:"-" mapstructure:"-"`
+	InheritMultipleFiles   bool          `yaml:"-" json:"-" mapstructure:"-"`
+	File                   string        `yaml:"-" json:"-" mapstructure:"-"`
```

Comment:
I left this here for compatibility, if only a single configuration file is desired. It needs to use `flags.StringVarP`, which needs to reference a string value, I'd also be happy to remove this.

Reply:
I'd lean on the fact that clio is pre 1.0 and make the breaking change